gradle/verification-metadata.xml

```diff
@@ -11,17 +11,17 @@
       </trusted-artifacts>
       <trusted-keys>
          <trusted-key id="0785b3eff60b1b1bea94e0bb7c25280eae63ebe5" group="^org[.]apache[.]httpcomponents($|([.].*))" regex="true"/>
+         <trusted-key id="2db4f1ef0fa761ecc4ea935c86fdc7e2a11262cb" group="commons-codec"/>
```

Comment:
What is the reason for moving those entries around?

Reply:
I'm assuming that they are now ordered in some way.

Reply:
Yeah, it looks like they are now ordered alphabetically by `id`.

Reply:
TESTING